_layouts/post.html

```diff
@@ -20,25 +20,27 @@
         <div class="container">
           <div class="row">
             <article>
-                <header class="page-header">
-                    <h1>{{ page.title }}</h1>
-                    <hr>
-                    <div class="author">
-                        <div class="author-media">
-                          <img src="/assets/images/blog/author.png" alt="Author picture">
-                        </div>
-                        <div class="author-content">
-                          <div class="author-name">{{page.author}}</div>
-                          <div class="article-date">{{ page.date | date: "%d.%m.%Y" }}
-                        </div>
-                        </div>
+              <header class="page-header">
+                <h1>{{ page.title }}</h1>
+                <div class="content-constrained">
```

Comment:
@khairahscorner this class="content-constrained" is new so perhaps that's causing the 'author block' width change?

Reply:
Yeah, it's part of the style updates I mentioned @lilyndk 
The author block taking the full width did not look too good imo, especially on larger screens. I think this looks better but we can get other opinions.

Reply:
Ok, yeah it does look unbalanced because we haven't implemented the 'full width' featured image and share functionality yet, as shown in the Figma design below. But if we change the design here, it needs to also be changed in the [related 'add share button' task](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/226) as well.

<img width="830" alt="Screenshot 2024-06-02 at 18 36 09" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/26842896/4d56e9f0-8e2a-4376-8696-805f876f70d9">

Reply:
My UI concern is that if we make the 'author block' much slimmer, it will look 'too cramped' next to the new share article button if the author has a long job title.

- But one way to solve that would be to remove the 'share' label from the share button and only display the round icon button in the slimmer 'author/share' header block.

Reply:
I think is fine for now. Let's focus more in the platform. For me that is really to merge :)